wip-mariocao-min-difficulty-dr.md

```diff
@@ -0,0 +1,213 @@
+
+Fig. 1 - Identities sorted by reputation in descending order.
+
+<br/>
+
+![Fig.2 - Application of the <i>eligibility algorithm</i> in previous sorted identities.](WIP-mariocao-min-difficulty-dr/eligibility.png "Fig.2 - Application of the <i>eligibility algorithm</i> in previous sorted identities.")
+
+Fig. 2 - Application of the <i>eligibility algorithm</i> in previous sorted identities.
+
+<br/>
+
+The _eligibility_ algorithm can be decomposed into 5 simple steps:
+
+1. Identities are sorted by reputation in descending order (see Fig.1).
+
+2. The upper edge of the trapezoid (the triangle) is calculated from the line following the equation `y = m*x + k`, where:
+
+    ```rust
+    let k = 1.5 * (average_reputation - minimum_reputation);
+    let m = -k / (active_reputed_identities_length - 1);
+    ```
+
+3. The curve is used to assign the initial eligibility to the previously sorted identities (see _Step 3: triangle_ in Fig.2).
+
+4. The identities eligibility is offset by adding the remaining reputation points that were not redistributed in the previous step (see _Step 4: rectangle_ in Fig.2).
+
+    ```rust
+    let remaining_reputation = total_active_reputation - total_triangle_reputation;
+    let offset_reputation = remaining_reputation / (active_reputed_identities_length);
+    ```
+
+    This step is important to keep the same data request mining difficulty for newcomers or Sybil identities.
+
+5. The rest of the `offset_reputation` integer division is assigned (in order) to the sorted identities until depleted (see _Step 5: remaining_ in Fig.2).
+
+    ```rust
+    let identities_with_extra_eligibility = (remaining_reputation) % active_reputed_identities_length;
+    ```
+
+Steps 4 and 5 aim to preserve the sum of reputation equal to the sum of eligibility. Geometrically, this can be defined as matching the area of the trapezoid (Fig.2) with the area of the reputation curve (Fig.1).
+
+The identity's reputation, from the perspective of chain state, is not altered after applying the [RepPoE] algorithm.
+
+
+## Specification
+
+
+### Removal of the dynamic threshold
+
+**(1)**: The [Dynamic Threshold] factor MUST be removed from the _RepPoE_ algorithm.
+
+As eligibility across reputed identities is fairer due to the changes described in [witnet-rust#1496], the [Dynamic Threshold] does not have anymore a significant impact on the probability of mining data requests. Additionally, the [Dynamic Threshold] was not properly updated to use the eligibility metrics instead of the reputation scores.
+
+In case of abnormally low ARS sizes, it is assumed that there are in the order of thousands of non-reputed identities computing their eligibility to be selected to solved the data request. This assumption is in line with the minimum difficulty set in [WIP-0012] and previous network conditions documented in [WIP-0009].
```

Comment:
```suggestion
In case of abnormally low ARS sizes, it is assumed that there is a number of non-reputed identities in the order of thousands computing their eligibility to be selected to solve the data request. This assumption is in line with the minimum difficulty set in [WIP-0012] and previous network conditions documented in [WIP-0009].
```